tests/testthat/test-shiny-snExplorer.R

```diff
@@ -1,12 +1,13 @@
+library(shinytest)
+library(testthat)
 context("app-file")
 # This file is for testing the applications in the inst/ directory.
 
-library(shinytest)
-
 test_that("snExplorer works", {
     skip_on_cran()
     skip_on_ci() # something's off on system dependencies
```

Comment:
Do we dare put test on through CI?

We're still in a branch, so this is the best timing we'll get.

Reply:
Do it! Isn't it the right time?